### ISSUE TITLE
Loss function in train

### ISSUE THREAD

**Issue by AllenPu** [xjtulyc/Ordinal_Regression_with_Multiple_Output_CNN_for_Age_Estimation#1]
Hi,
I noticed that the train used the MSE for loss while the paper states the CE for train, is that OK?
